### ISSUE TITLE
Field 306 and bf:duration

### ISSUE THREAD

**Issue by kiegel** [lcnetdev/marc2bibframe2#58]
The semantics of field 306 (Playing time) can cause problems.  Everything is fine when 306 contains a single value for the duration of the Instance as a whole.  There is a problem, however, when the Instance contains two or more pieces, in which case field 306 contains multiple values, one for each part.

Take this example:
bf:duration "000213",
        "000215",
        "000312",
        "000344",
        "000347",
        "000354",
        "000739",
        "000830",
        "003121" ;
    bf:extent [ a bf:Extent ;
            rdfs:label "1 audio disc (70 min., 30 sec.)" ] .

This linked data makes the nonsensical assertion that the Instance has nine separate and different durations.  In fact, it has one duration, 70 minutes and 30 seconds.  The durations of parts logically belong in Instances for these parts, which are not generated in the conversion.

I suggest that when field 306 has multiple values, bf:duration not be generated.  Alternately, you could create a new property, bf:durationOfPart, that would give the proper meaning to the values.

**Comment by wafschneider** [comment]
Thanks, @kiegel ! I agree, that doesn't look right. Could you please attach the source MARC record, so we have a concrete example to work from?

**Comment by kiegel** [comment]
Attached is the record I used, in MARC XML.

Joe

**Comment by wafschneider** [comment]
Here's the record:
```
<?xml version="1.0" encoding="UTF-8"?>
<marc:collection xmlns:marc="http://www.loc.gov/MARC21/slim" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.loc.gov/MARC21/slim http://www.loc.gov/standards/marcxml/schema/MARC21slim.xsd">
  <marc:record>
    <marc:leader>04143cjm a2200721Ii 4500</marc:leader>
    <marc:controlfield tag="001">99160577470001452</marc:controlfield>
    <marc:controlfield tag="005">20141002054150.1</marc:controlfield>
    <marc:controlfield tag="007">sd fsngnn|||ed</marc:controlfield>
    <marc:controlfield tag="008">130315p20132012enkmunn  defi     n lat d</marc:controlfield>
    <marc:datafield tag="024" ind1="1" ind2="0">
      <marc:subfield code="a">828021610622</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="028" ind1="0" ind2="2">
      <marc:subfield code="a">COR16106</marc:subfield>
      <marc:subfield code="b">Coro</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="035" ind1=" " ind2=" ">
      <marc:subfield code="a">(OCoLC)830033400</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="035" ind1=" " ind2=" ">
      <marc:subfield code="a">(OCoLC)ocn830033400</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="040" ind1=" " ind2=" ">
      <marc:subfield code="a">Nz</marc:subfield>
      <marc:subfield code="b">eng</marc:subfield>
      <marc:subfield code="e">rda</marc:subfield>
      <marc:subfield code="c">NZWPM</marc:subfield>
      <marc:subfield code="d">OCLCO</marc:subfield>
      <marc:subfield code="d">CIN</marc:subfield>
      <marc:subfield code="d">OCLCQ</marc:subfield>
      <marc:subfield code="d">TEF</marc:subfield>
      <marc:subfield code="d">OCLCF</marc:subfield>
      <marc:subfield code="d">MYG</marc:subfield>
      <marc:subfield code="d">TOH</marc:subfield>
      <marc:subfield code="d">VRC</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="041" ind1="0" ind2=" ">
      <marc:subfield code="d">lat</marc:subfield>
      <marc:subfield code="e">lat</marc:subfield>
      <marc:subfield code="e">eng</marc:subfield>
      <marc:subfield code="n">lat</marc:subfield>
      <marc:subfield code="g">eng</marc:subfield>
      <marc:subfield code="g">ita</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="047" ind1=" " ind2=" ">
      <marc:subfield code="a">ms</marc:subfield>
      <marc:subfield code="a">mo</marc:subfield>
      <marc:subfield code="a">hy</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="049" ind1=" " ind2=" ">
      <marc:subfield code="a">MAIN</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="050" ind1=" " ind2="4">
      <marc:subfield code="a">M2082.P35</marc:subfield>
      <marc:subfield code="b">G56 2013 v.3</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="082" ind1="0" ind2="4">
      <marc:subfield code="a">782.2/2</marc:subfield>
      <marc:subfield code="2">23</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="100" ind1="1" ind2=" ">
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594,</marc:subfield>
      <marc:subfield code="e">composer.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="240" ind1="1" ind2="0">
      <marc:subfield code="a">Vocal music.</marc:subfield>
      <marc:subfield code="k">Selections</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="245" ind1="1" ind2="0">
      <marc:subfield code="a">Giovanni Pierluigi da Palestrina.</marc:subfield>
      <marc:subfield code="n">Volume 3.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="246" ind1="1" ind2=" ">
      <marc:subfield code="i">Title on container spine:</marc:subfield>
      <marc:subfield code="a">Palestrina.</marc:subfield>
      <marc:subfield code="n">Volume 3</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="264" ind1=" " ind2="1">
      <marc:subfield code="a">[London] :</marc:subfield>
      <marc:subfield code="b">Coro,</marc:subfield>
      <marc:subfield code="c">[2013]</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="264" ind1=" " ind2="4">
      <marc:subfield code="c">℗2013</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="300" ind1=" " ind2=" ">
      <marc:subfield code="a">1 audio disc (70 min., 30 sec.) :</marc:subfield>
      <marc:subfield code="b">digital, CD audio, stereo. ;</marc:subfield>
      <marc:subfield code="c">4 3/4 in.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="306" ind1=" " ind2=" ">
      <marc:subfield code="a">000830</marc:subfield>
      <marc:subfield code="a">000739</marc:subfield>
      <marc:subfield code="a">000312</marc:subfield>
      <marc:subfield code="a">000344</marc:subfield>
      <marc:subfield code="a">000347</marc:subfield>
      <marc:subfield code="a">000354</marc:subfield>
      <marc:subfield code="a">000347</marc:subfield>
      <marc:subfield code="a">000215</marc:subfield>
      <marc:subfield code="a">000213</marc:subfield>
      <marc:subfield code="a">003121</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="336" ind1=" " ind2=" ">
      <marc:subfield code="a">performed music</marc:subfield>
      <marc:subfield code="b">prm</marc:subfield>
      <marc:subfield code="2">rdacontent</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="337" ind1=" " ind2=" ">
      <marc:subfield code="a">audio</marc:subfield>
      <marc:subfield code="b">s</marc:subfield>
      <marc:subfield code="2">rdamedia</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="338" ind1=" " ind2=" ">
      <marc:subfield code="a">audio disc</marc:subfield>
      <marc:subfield code="b">sd</marc:subfield>
      <marc:subfield code="2">rdacarrier</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="344" ind1=" " ind2=" ">
      <marc:subfield code="a">digital</marc:subfield>
      <marc:subfield code="g">stereo</marc:subfield>
      <marc:subfield code="2">rda</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="347" ind1=" " ind2=" ">
      <marc:subfield code="a">audio file</marc:subfield>
      <marc:subfield code="b">CD audio</marc:subfield>
      <marc:subfield code="2">rda</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="500" ind1=" " ind2=" ">
      <marc:subfield code="a">Title from disc label.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="500" ind1=" " ind2=" ">
      <marc:subfield code="a">Compact disc.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="500" ind1=" " ind2=" ">
      <marc:subfield code="a">Program notes in English and Italian and texts with English translations (33 p.) in container.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="505" ind1="0" ind2=" ">
      <marc:subfield code="a">Stabat Mater : a 8 (8:30) -- Ad caenam Agni providi : hymn (7:39) -- Song of Songs: nos. 4-6. Vineam meam non custodivi (3:12) ; Si ignoras te, O pulchra (3:44) ; Pulchrae sunt genae tuae (3:47) -- Regina caeli : a 8 (3:54) -- Three offertories for Easter period. Improperium expectavit cor meum (3:47) ; Confitebor tibi Domine (2:15) ; Terra tremuit (2:13) -- Missa Regina caeli (31:21).</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="511" ind1="0" ind2=" ">
      <marc:subfield code="a">The Sixteen ; Harry Christophers, conductor.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="518" ind1=" " ind2=" ">
      <marc:subfield code="o">Recorded</marc:subfield>
      <marc:subfield code="d">2012 October</marc:subfield>
      <marc:subfield code="p">Church of St. Alban the Martyr, Holborn, London.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="520" ind1=" " ind2=" ">
      <marc:subfield code="a">Harry Christophers and The Sixteen continue their exploration of his work with a disc of music for the Easter period. Many of the works on this new recording celebrate the joyful part of Easter: the Resurrection and the central mass is the wonderfully inspired Missa Regina caeli.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="546" ind1=" " ind2=" ">
      <marc:subfield code="a">Sung in Latin.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="0">
      <marc:subfield code="a">Masses, Unaccompanied.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="0">
      <marc:subfield code="a">Choruses, Sacred (Mixed voices), Unaccompanied.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="0">
      <marc:subfield code="a">Motets.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="0">
      <marc:subfield code="a">Offertories (Music)</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="0">
      <marc:subfield code="a">Hymns, Latin.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="7">
      <marc:subfield code="a">Choruses, Sacred (Mixed voices), Unaccompanied.</marc:subfield>
      <marc:subfield code="2">fast</marc:subfield>
      <marc:subfield code="0">(OCoLC)fst00858730</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="7">
      <marc:subfield code="a">Hymns, Latin.</marc:subfield>
      <marc:subfield code="2">fast</marc:subfield>
      <marc:subfield code="0">(OCoLC)fst00965595</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="7">
      <marc:subfield code="a">Masses, Unaccompanied.</marc:subfield>
      <marc:subfield code="2">fast</marc:subfield>
      <marc:subfield code="0">(OCoLC)fst01011524</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="7">
      <marc:subfield code="a">Motets.</marc:subfield>
      <marc:subfield code="2">fast</marc:subfield>
      <marc:subfield code="0">(OCoLC)fst01026871</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="650" ind1=" " ind2="7">
      <marc:subfield code="a">Offertories (Music)</marc:subfield>
      <marc:subfield code="2">fast</marc:subfield>
      <marc:subfield code="0">(OCoLC)fst01044051</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2=" ">
      <marc:subfield code="a">Christophers, Harry,</marc:subfield>
      <marc:subfield code="e">conductor.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Stabat Mater,</marc:subfield>
      <marc:subfield code="m">voices (8)</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Hymni totius anni.</marc:subfield>
      <marc:subfield code="p">Ad coenam Agni providi.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Motets</marc:subfield>
      <marc:subfield code="n">(1583).</marc:subfield>
      <marc:subfield code="p">Vineam meam.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Motets</marc:subfield>
      <marc:subfield code="n">(1583).</marc:subfield>
      <marc:subfield code="p">Si ignoras te.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Motets</marc:subfield>
      <marc:subfield code="n">(1583)</marc:subfield>
      <marc:subfield code="p">Pulchrae sunt genae.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Regina coeli,</marc:subfield>
      <marc:subfield code="m">voices (8),</marc:subfield>
      <marc:subfield code="n">no. 1.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Offertories,</marc:subfield>
      <marc:subfield code="m">voices (5).</marc:subfield>
      <marc:subfield code="p">Improperium.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Offertories,</marc:subfield>
      <marc:subfield code="m">voices (5).</marc:subfield>
      <marc:subfield code="p">Confitebor tibi.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Offertories,</marc:subfield>
      <marc:subfield code="m">voices (5).</marc:subfield>
      <marc:subfield code="p">Terra tremuit.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="700" ind1="1" ind2="2">
      <marc:subfield code="i">Contains (work):</marc:subfield>
      <marc:subfield code="a">Palestrina, Giovanni Pierluigi da,</marc:subfield>
      <marc:subfield code="d">1525?-1594.</marc:subfield>
      <marc:subfield code="t">Masses,</marc:subfield>
      <marc:subfield code="n">book 11.</marc:subfield>
      <marc:subfield code="p">Missa Regina Coeli.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="710" ind1="2" ind2=" ">
      <marc:subfield code="a">Sixteen (Musical group),</marc:subfield>
      <marc:subfield code="e">performer.</marc:subfield>
    </marc:datafield>
    <marc:datafield tag="994" ind1=" " ind2=" ">
      <marc:subfield code="a">92</marc:subfield>
      <marc:subfield code="b">OCACL</marc:subfield>
    </marc:datafield>
  </marc:record>
</marc:collection>
```

**Comment by jodiw01** [comment]
Specifications have been updated.

**Comment by kirkhess** [comment]
Another example: http://hdl.handle.net/10079/bibid/1711688
the 008 value needs to have this format: PTnnnM (should be in specs...)
<bf:duration rdf:datatype="http://www.w3.org/2001/XMLSchema#duration">PT105M</bf:duration>

**Comment by jodiw01** [comment]
Specs are updated. Conversion of 306 tag is included in v1.4. Conversion of 008 tag will be refined in v1.6
